list-all-pages; mailgun lib replace

DBLaci opened the pull request:
The changes I made:

- mailgun lib is deprecated - I changed it to the official one - the configuration hasn't changed
- we search all pages until empty found - this is the goal of the feature
- minor changes in formatting and readme

icebob reviewed (commented):
Good job, thanks!, Did you tested it on mailgun with the new lib?

DBLaci commented:
FIrst: I updated the Dockerfile in this commit, because instead of clone I used add, but adding everything was not safe and sane, sorry.

Second: yes, I tested. Maybe the originally used lib wasn't bad, but I added wrong url to the config. BUT the old lib was wrong because it said the sending was fine - which was a lie... SO in the lib I added the errors are handled correctly... And yes it works for me!

icebob commented:
Great, thanks!
